package.json

```
@@ -29,21 +30,20 @@
     "e2e"
   ],
   "files": [
-    "dist",
+    "src/pure.js",
+    "src/index.js",
```

I would make this `src/` so that things don't unexpectedly break if you add another file
```suggestion
    "src/",
```

---

Point. Done!